### PR TITLE
Add copy button to glyph pages

### PR DESCRIPTION
**Pull request by andyzito**

### CODE REVIEW

File: main/components/ClipboardCopy/ClipboardCopy.js [hunk 1]
```diff
@@ -44,12 +70,17 @@ const StyledTip = styled.span`
   border-radius: 0.5em;
 `
 
+const StyledIconPosition = `
+`
+
```
Comment (kwiliarty) [inline main/components/ClipboardCopy/ClipboardCopy.js:75]: I think we can should just remove `StyledIconPosition`.

File: main/components/ClipboardCopy/ClipboardCopy.js [hunk 2]
```diff
@@ -7,6 +7,32 @@ import styled from 'styled-components'
...
+  position:relative;
+  background-color: transparent;
+  background-clip: padding-box;
+  color: ${ props => props.theme.colors.darkRed };
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  column-gap: 0.5rem;
+  height: 2.2rem;
```
Comment (kwiliarty) [inline main/components/ClipboardCopy/ClipboardCopy.js:19]: Let's change these `rem`s to `em`s. I think these values should depend on the current font size. I know that I put one of them in there, but I'm just re-thinking it.

File: main/components/ClipboardCopy/ClipboardCopy.js [hunk 3]
```diff
@@ -44,12 +70,17 @@ const StyledTip = styled.span`
   border-radius: 0.5em;
 `
 
+const StyledIconPosition = `
+`
+
 const StyledContentCopyRoundedIcon = styled( ContentCopyRoundedIcon )`
   color: inherit;
+  ${ StyledIconPosition }
```
Comment (kwiliarty) [inline main/components/ClipboardCopy/ClipboardCopy.js:78]: This `StyledIconPosition` can go.

File: main/components/ClipboardCopy/ClipboardCopy.js [hunk 4]
```diff
@@ -44,12 +70,17 @@ const StyledTip = styled.span`
...
 `
 
 const StyledCheckRoundedIcon = styled( CheckRoundedIcon )`
-  color: ${ props => props.theme.colors.darkGreen };
+  color: inherit;
+  ${ StyledIconPosition }
```
Comment (kwiliarty) [inline main/components/ClipboardCopy/ClipboardCopy.js:83]: This `StyledIconPosition` can go.

File: main/components/ClipboardCopy/ClipboardCopy.js [hunk 5]
```diff
@@ -87,11 +121,17 @@ ClipboardCopy.propTypes = {
   hint: PropTypes.string,
   /** The contents to be copied to the clipboard */
   text: PropTypes.string,
+  /** Button or icon */
+  button: PropTypes.bool,
+  /** A background color name */
```
Comment (kwiliarty) [inline main/components/ClipboardCopy/ClipboardCopy.js:126]: I think the comment should be "A text color name"

**Review by kwiliarty — CHANGES_REQUESTED** [review]
I've requested some small changes here. Besides that, could you add something to the jest testing for this component? There's an example of a similar test in `main/components/IpaPairedInput/IpaPairedInput.test.js`. The tests are in an interesting space between unit tests and end-to-end tests. Let me know if you want to chat about it.

**Comment by kwiliarty** [comment]
Oh, I just noticed that this is actually still a draft. Sorry if I jumped in too soon. Anyway, maybe the feedback is still helpful?

**Comment by andyzito** [comment]
Not at all, thank you for catching those things. They should all be resolved in the latest commit.

**Comment by andyzito** [comment]
@kwiliarty Is this what you had in mind for tests? Not 100% sure the `expectToBeCalled` lines are correct, but the tests pass.

**Review by kwiliarty — APPROVED** [review]
That looks great, and the test is right. It just ensures that the click calls the intended function with the intended argument. Thanks so much for this!

**Comment by kwiliarty** [comment]
@andyzito : This is still marked as a draft. Was there something else you wanted to do on it, or do you think it's ready to go?

**Comment by andyzito** [comment]
@kwiliarty Ready!

**Pull request closed without merging**